--- nginx.spec.original	2016-04-21 16:25:16.196478925 +0200
+++ nginx.spec.patched	2016-04-21 16:32:30.867620160 +0200
 
@@ -121,9 +121,11 @@
         --with-mail_ssl_module \
         --with-file-aio \
         --with-ipv6 \
+        --add-module=/nginx-lua/ngx_devel_kit-0.3.0rc1 \
+        --add-module=/nginx-lua/lua-nginx-module-0.10.2 \
         %{?with_http2:--with-http_v2_module}")
 
-Summary: High performance web server
+Summary: High performance web server (reBuy)
 Name: nginx
 Version: %{main_version}
 Release: %{main_release}
